### ISSUE TITLE
ValueError: Unable to construct build path with source

### ISSUE THREAD

**Issue by audachang** [poldracklab/fmriprep#2004]
Running fmriprep-docker on a ubuntu 19.10 platform with 24 cpus and 64GB RAM. 

With the following commands:
`fmriprep-docker proc_data proc_data/derivatives participant --participant-label TEST99 --fs-license-file $FREESURFER_HOME/license.txt`

After the processing going on for a while and having created most of the anatomical and functional outputs in the derivatives/fmriprep folder, came the error messages attached below. The crash log in the log folder is also attached. Any hint to resolve it would be appreciated. 

> [Node] Finished "fmriprep_wf.single_subject_TEST99_wf.func_preproc_exe_img_rest_run_01_wf.bold_std_trans_wf.bold_reference_wf.enhance_and_skullstrip_bold_wf.apply_mask".

> Traceback (most recent call last):
> File "/usr/local/miniconda/bin/fmriprep", line 10, in <module>   sys.exit(main())
> File "/usr/local/miniconda/lib/python3.7/site-packages/fmriprep/cli/run.py", line 465, in main
    packagename='fmriprep')
> File "/usr/local/miniconda/lib/python3.7/site-packages/niworkflows/reports/core.py", line 478, in generate_reports
    for subject_label in subject_list
  File "/usr/local/miniconda/lib/python3.7/site-packages/niworkflows/reports/core.py", line 478, in <listcomp>
    for subject_label in subject_list
  File "/usr/local/miniconda/lib/python3.7/site-packages/niworkflows/reports/core.py", line 465, in run_reports
    subject_id=subject_label, packagename=packagename)
  File "/usr/local/miniconda/lib/python3.7/site-packages/niworkflows/reports/core.py", line 267, in __init__
    self._load_config(Path(config))
  File "/usr/local/miniconda/lib/python3.7/site-packages/niworkflows/reports/core.py", line 286, in _load_config
    self.index(settings['sections'])
  File "/usr/local/miniconda/lib/python3.7/site-packages/niworkflows/reports/core.py", line 325, in index
    rlet = Reportlet(self.layout, self.out_dir, config=cfg)
  File "/usr/local/miniconda/lib/python3.7/site-packages/niworkflows/reports/core.py", line 164, in __init__
    linked_svg = layout.build_path(entities, validate=False)
  File "/usr/local/miniconda/lib/python3.7/site-packages/bids/layout/layout.py", line 1351, in build_path
    raise ValueError("Unable to construct build path with source {}".format(source))
ValueError: Unable to construct build path with source {'datatype': 'figures', 'desc': 'bbregister', 'run': 1, 'subject': 'TEST99', 'suffix': 'bold', 'extension': 'svg'}
Sentry is attempting to send 0 pending error messages
Waiting up to 2 seconds
Press Ctrl-C to quit
>fMRIPrep: Please report errors to https://github.com/poldracklab/fmriprep/issues

[crash-20200226-202341-root-summary-28e8a5a0-92d9-4b21-8fe1-2b2a5b3ce076.txt](https://github.com/poldracklab/fmriprep/files/4259077/crash-20200226-202341-root-summary-28e8a5a0-92d9-4b21-8fe1-2b2a5b3ce076.txt)

**Comment by effigies** [comment]
What version of fMRIPrep are you using?

**Comment by audachang** [comment]
my fMRIPrep version is v1.5.9.

**Comment by effigies** [comment]
Thanks for the report. Seeing this in another project as well, so hopefully it won't be too hard to track down exactly why it's failing.

**Comment by oesteban** [comment]
@audachang, are your inputs BIDS-compliant? (spoiler: they are not).

I've checked on your attached crashfile and noticed that the first error happened on a different (earlier) step of fMRIPrep. Looking at the inputs of that step I find:

```
bold = ['/data/sub-TEST99/func/sub-TEST99_exe-img-rest_run-01_bold.nii.gz', '/data/sub-TEST99/func/sub-TEST99_exe-img-rest_run-02_bold.nii.gz']
```

your files have some spurious `exe-img-` string - if you replace `s/exe-img/task/g` your dataset will be then valid.

Closing and removing the bug label.
